IOTConnectSDK/iotconnect-sdk/build.gradle

```diff
@@ -62,8 +62,8 @@ afterEvaluate {
                 // You can then customize attributes of the publication as shown below.
                 groupId = 'com.github.sw-ms-imranbhadelia'
                 artifactId = 'iotconnect-sdk'
-                version = '3.1.3'
+                version = '30.0.0'
```

Comment:
Why there is a big difference between 3.1.3 and 30.0.0?

Reply:
I have changed the version but it is of old iot connect sdk i.e. 1.0 remain commented only when we create library for new iot connect sdk i.e. 2.0